chia/util/keyring_wrapper.py

```diff
@@ -104,24 +106,37 @@ class KeyringWrapper:
     cached_passphrase_is_validated: bool = False
     legacy_keyring = None
 
-    def __init__(self, keys_root_path: Path = DEFAULT_KEYS_ROOT_PATH, force_legacy: bool = False):
+    @classmethod
+    def create(
+        cls: Type[_T_KeyringWrapper], keys_root_path: Path = DEFAULT_KEYS_ROOT_PATH, force_legacy: bool = False
+    ) -> Optional[_T_KeyringWrapper]:
```

Comment:
I thought the main reason to have a `create()` function (instead of a constructor) is that it can be `async`. But this isn't. @altendky has also pointed out that with a `create()` function you can mark the class as `@dataclass` (which has some benefits I haven't quite understood yet). But this class isn't a `dataclass` either.

Reply:
You know, I had assumed that the reason for the create method was so that we could avoid a global, and instead construct at a later time of our choosing, but I don't know. @altendky @paninaro ?

Reply:
I honestly don't remember why I had added the create method here... Looking at where it's used and what's checked into main, I don't see any functional difference between `create()` and `__init__()`. It may have been that I was trying to convey that multiple KeyringWrappers shouldn't be created.

Reply:
It would seem reasonable to not return an `Optional[]`, but just a `_T_KeyringWrapper` directly. If there's an error, throw an exception. This would fix the mypy error